src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -21,28 +26,65 @@
  */
 public class AddCommandParser implements Parser<AddCommand> {
 
+    /**
+     * Throws a ParseException if the type is not specified correctly in the input arguments.
+     * If the type is specified correctly, it returns the type.
```

Comment:
ur docs need to have @param and @return i think?

Reply:
Rectified in commit "Improve coding style of AddCommandParser.java"